src/chi/components/steps/steps.scss

```diff
@@ -0,0 +1,86 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  .a-steps {
+    display: table;
+    padding: 0;
+    table-layout: fixed;
+    width: 100%;
+
+    & > li {
+      color: $brand-color;
+      display: table-cell;
+      font-size: $text;
+      line-height: 1rem;
+      position: relative;
+      text-align: center;
+      z-index: 0;
+
+      &::before {
```

Comment:
Change to:

```
&::before {
  background-color: $brand-color;
  border: 0.125rem solid $text-white;
  border-radius: 50%;
  content: '';
  display: block;
  height: 1rem;
  line-height: 1rem;
  margin: 0 auto 0.5rem;
  width: 1rem;
}
```

Reply:
I changed with this code and now not is horizontal centered i think. what do you think? It 
<img width="919" alt="screen shot 2018-11-21 at 9 52 24 am" src="https://user-images.githubusercontent.com/43439742/48829973-d8419a00-ed73-11e8-894f-6e7e53d1d2b9.png">
is ok?

Reply:
@davidojedacl Good catch. Can we fix this while ensuring the circle is rendered as a shape versus a web font?